### ISSUE TITLE
Create a popup 

### ISSUE THREAD

**Issue by abudhakeer** [yosevu/react-content-script#1]
Hey Yosevu,
Thank you for creating this awesome template. It really helped me in building chrome extensions using react. Though there are multiple examples out there but i couldn't find any with content scripts. Only your example helped me with it.

But when i tried to add a popup to my extension i face some trouble. Because, in your example we consider `App.js` as content script. But if i use `index.html` as popup how can i use `App.js` as content script?

Basically, can you please help in sharing an example for using both popup and content script?

**Comment by yosevu** [comment]
Hi @abudhakeer, I'm not sure how much the build would have to change, but will try to take a look at adding a popup example too.

**Comment by kasparas12** [comment]
@yosevu any chance you've played with that example? It would also be really needed for me. Thanks.

**Comment by yosevu** [comment]
I updated the build to work for Chrome Extension Content Scripts and Popups @abudhakeer @kasparas12. Let me know if you run into any issues!
